### ISSUE TITLE
[BUG] Error in Fabric 1.18.2 server

### ISSUE THREAD

**Issue by pavog** [TheFizzWare/DramaticDoors#13]
**Important Information**
Minecraft Version: **1.18.2**
Fabric Version: **0.14.6**
DramaticDoors Version: **[DramaticDoors-FABRIC-1.18.2-1.0.jar](https://www.curseforge.com/minecraft/mc-mods/dramatic-doors/files/3802244)**

**Describe the bug**
I created a brand new Fabric 1.18.2 server, installed fabric api and DramaticDoors.
When I start the server it throws some exceptions and writes them to my server log.
However the server does not crash.

Error message:
```
[Worker-Main-4/ERROR]: Couldn't read tag list dramaticdoors:tall_doors from dramaticdoors:tags/blocks/tall_doors.json in data pack Dramatic Doors (Fabric)
com.google.gson.JsonParseException: com.google.gson.stream.MalformedJsonException: Use JsonReader.setLenient(true) to accept malformed JSON at line 6 column 4 path $.values[2]
```

**To Reproduce**
* Download Fabric 1.18.2 / 0.14.6
* Download Fabric API
* Download DramaticDoors Fabric
* Start the server
* See the log

**Expected behavior**
No errors ;-)

Full error log:
https://mclo.gs/6Si9tXZ

**Comment by Kitteh6660** [comment]
Should hopefully be fixed in mod version 1.1 for Fabric!

**Comment by pavog** [comment]
Yes, I can confirm that these errors do not occur in the latest release [DramaticDoors-FABRIC-1.18.2-1.1.jar](https://www.curseforge.com/minecraft/mc-mods/dramatic-doors/files/3811603).
